Makefile

```diff
@@ -672,6 +672,222 @@ tracker_dump_db:
+              --vm_program ./scripts/test_files/risc_zero/fibonacci_proof_generator/fibonacci_id.bin \
+              --public_input ./scripts/test_files/risc_zero/fibonacci_proof_generator/risc_zero_fibonacci.pub \
+              --repetitions $(DOCKER_BURST_SIZE) \
+              --proof_generator_addr $(PROOF_GENERATOR_ADDRESS) \
+              --rpc_url $(DOCKER_RPC_URL)
+
+docker_batcher_send_plonk_bn254_burst:
+	@echo "Sending Groth16Bn254 1!=0 task to Batcher..."
+	docker exec $(shell docker ps | grep batcher | awk '{print $$1}') aligned submit \
+              --private_key $(DOCKER_PROOFS_PRIVATE_KEY) \
+              --batcher_url 'ws://[::1]:8080' \
+              --proving_system GnarkPlonkBn254 \
+              --proof ./scripts/test_files/gnark_plonk_bn254_script/plonk.proof \
+              --public_input ./scripts/test_files/gnark_plonk_bn254_script/plonk_pub_input.pub \
+              --vk ./scripts/test_files/gnark_plonk_bn254_script/plonk.vk \
+              --proof_generator_addr $(PROOF_GENERATOR_ADDRESS) \
+              --rpc_url $(DOCKER_RPC_URL) \
+              --repetitions $(DOCKER_BURST_SIZE)
+
+docker_batcher_send_plonk_bls12_381_burst:
+	@echo "Sending Groth16 BLS12-381 1!=0 task to Batcher..."
+	docker exec $(shell docker ps | grep batcher | awk '{print $$1}') aligned submit \
+              --private_key $(DOCKER_PROOFS_PRIVATE_KEY) \
+              --batcher_url 'ws://[::1]:8080' \
+              --proving_system GnarkPlonkBls12_381 \
+              --proof ./scripts/test_files/gnark_plonk_bls12_381_script/plonk.proof \
+              --public_input ./scripts/test_files/gnark_plonk_bls12_381_script/plonk_pub_input.pub \
+              --vk ./scripts/test_files/gnark_plonk_bls12_381_script/plonk.vk \
+              --proof_generator_addr $(PROOF_GENERATOR_ADDRESS) \
+              --repetitions $(DOCKER_BURST_SIZE) \
+              --rpc_url $(DOCKER_RPC_URL)
+
+docker_batcher_send_groth16_burst:
+	@echo "Sending Groth16 BLS12-381 1!=0 task to Batcher..."
+	docker exec $(shell docker ps | grep batcher | awk '{print $$1}') aligned submit \
+              --private_key $(DOCKER_PROOFS_PRIVATE_KEY) \
+							--proving_system Groth16Bn254 \
+							--proof ./scripts/test_files/gnark_groth16_bn254_script/groth16.proof \
+							--public_input ./scripts/test_files/gnark_groth16_bn254_script/plonk_pub_input.pub \
+							--vk ./scripts/test_files/gnark_groth16_bn254_script/groth16.vk \
+							--proof_generator_addr $(PROOF_GENERATOR_ADDRESS) \
+  						--repetitions $(DOCKER_BURST_SIZE) \
+							--rpc_url $(DOCKER_RPC_URL)
+
+# Update target as new proofs are supported.
+docker_batcher_send_all_proofs_burst:
+	@$(MAKE) docker_batcher_send_sp1_burst
+	@$(MAKE) docker_batcher_send_risc0_burst
+	@$(MAKE) docker_batcher_send_plonk_bn254_burst
+	@$(MAKE) docker_batcher_send_plonk_bls12_381_burst
+	@$(MAKE) docker_batcher_send_groth16_burst
+
+docker_batcher_send_infinite_groth16:
```

Comment:
Any reason  why this is not a .sh script? I saw you made an issue, I think it would be best to migrate it now and avoid increasing the technical debt so much. Anyhow it should be pretty straight forward

Reply:
We don't want to delay merging this anymore. We need the CI working ASAP. I can do it with high priority if needed, but doing it in this PR would take another whole feedback loop with the review/test approval and such, and we've seen that takes some time.

Reply:
FYI https://github.com/yetanotherco/aligned_layer/issues/1240